xyz

```diff
@@ -134,10 +134,11 @@ if type tput &>/dev/null ; then
   reset=$(tput sgr0)
 fi
 
-printf "Current version is ${bold}${version}${reset}. "
-printf "Press [enter] to publish ${bold}${name}@${next_version}${reset}."
```

Comment:
```bash
printf "Current version is ${bold}${version}${reset}. "
       ^-- SC2059
printf "Press [enter] to publish ${bold}${name}@${next_version}${reset}."
       ^-- SC2059
```

[SC2059][1]: Don't use variables in the `printf` format string. Use `printf "..%s.." "$foo"`.


[1]: https://github.com/koalaman/shellcheck/wiki/SC2059